### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ashutosh-bisoyi**
Updated readme file with a meaningful basic content.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,66 @@
...
+
+## Overview
+
+The Dev Standards Repository is a collection of detailed guides, each catering to a specific programming language or technology. Our mission is to create a hub of knowledge that empowers developers to follow consistent coding standards, set up efficient workflows, and embrace best practices.
+
+## Languages Covered
+
+We provide guides for a diverse array of languages and technologies, including but not limited to:
+
+- Power Apps
```
Comment (osm-vishnukyatannawar) [inline README.md:23]: Can we make them links?

Comment (osm-vishnukyatannawar) [reply README.md:23]: Also for now only keep the following

- These are the only ones which we published properly other we still have to work on them

- UI Guidelines
- Angular
- CSS
- Database
- Dotnet
- Flutter
- HTML
- Log Format
- PHP

File: README.md [hunk 2]
```diff
@@ -1,2 +1,66 @@
...
-Documentation for all the development category standards followed at Osmosys
+# Osmosys Dev Standards
+
+Welcome to the Osmosys's Dev Standards Repository! This open-source project is a cornerstone of our commitment to excellence in software development. We've curated an extensive collection of coding standards, CI setup guides, and best practices spanning a multitude of programming languages. By collaborating here, you're contributing to the development community's efforts to streamline workflows, improve code quality, and promote best practices.
+
+## Table of Contents
+
+- [Osmosys Dev Standards](#osmosys-dev-standards)
+  - [Table of Contents](#table-of-contents)
+  - [Overview](#overview)
```
Comment (osm-vishnukyatannawar) [inline README.md:9]: May be consider adding 2 sections one for coding standards and one for CI setups and also giving list to CI setup documents.

**Review by osm-vishnukyatannawar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ashutosh-bisoyi) [reply README.md]: Yes, it's a good idea to make them links.
I have updated the list and added corresponding links to them

File: README.md (hunk 2 above)
Comment (ashutosh-bisoyi) [reply README.md]: Updated the doc to have

1. Repo scope - this explains what all we offer right now
2. Individual details - based on scope we explains individual overview and languages covered

**Review by osm-vishnukyatannawar — APPROVED** [review]

**Pull request merged**